### PR TITLE
`Addresses` field in watch list data contains empty objects

### PR DESCRIPTION
**Pull request by chaitanya11**
This issue is because `jackson` is not able to map result to object. Mapping properties added.(`@jsonporperty`)

### CODE REVIEW

File: src/main/java/com/blockscore/models/Address.java [hunk 1]
```diff
@@ -183,4 +191,6 @@ public String getPostalCode() {
   public String getCountryCode() {
     return countryCode;
   }
+
 }
+
```
Comment (chaitanya11) [inline src/main/java/com/blockscore/models/Address.java:196]: @dgollahon  without `@jsonProperty` Address object cannot be created from json response while searching watch list. Empty objects will be created for address.

Comment (dgollahon) [reply src/main/java/com/blockscore/models/Address.java:196]: Hey @chaitanya11, sorry for the delay and thanks for the reminder ping. I should be able to review/audit this later today or tomorrow.

Comment (archana-na) [reply src/main/java/com/blockscore/models/Address.java:196]: @dgollahon Is this request being reviewed?.

Comment (dgollahon) [reply src/main/java/com/blockscore/models/Address.java:196]: @archana-na: Hey, sorry for the delay again. I had some other tasks that took precedence over this before the long weekend, but I will be able to review it sometime today for certain.

**Comment by dgollahon** [comment]
Hey @chaitanya11, I cherry-picked your changes from here and updated some tests, annotations, and formatting in #7. I'll do a release shortly. Thanks for catching/fixing this!

**Pull request closed without merging**